lib/facter/tuned.active.rb

```diff
@@ -0,0 +1,49 @@
+require 'facter'
+
+def get_active_profile
+  active_profile = ""
```

Comment:
I prefer consistent use of single & double quoting, and for me use of single quote denotes a simple string and use of double quotes indicates that something is going to be interpolated inside the string.  So I think you can fix this just by replacing all double quotes with single quotes.